README.md

```diff
@@ -125,7 +125,7 @@ You can pass additional options to CMake to customize the build.
 Important options are
 
 - `-G Ninja` to use the Ninja backend instead of the default Make backend. Other build backends are available with a similar syntax.
-- `-DCMAKE_INSTALL_PREFIX` is used to provide the install location for the library.
+- `-DCMAKE_INSTALL_PREFIX` is used to provide the install location for the library. If not provided, this defaults to `stdlib_install` in the `stdlib` directory.
```

Comment:
In fact the default is platform dependent, *e.g.* `/usr/local` on Unix and `c:/Program Files/${PROJECT_NAME}` on Windows. See https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html

Reply:
Oops, thank you